delta-proto/src/main/java/io/cdap/delta/proto/ColumnTransformation.java

```diff
@@ -67,5 +69,10 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(columnName, directive);
   }
+
+  @Override
+  public String toString() {
+    return String.format("ColumnTransformation(columnName : \"%s\", directive : \"%s\")", columnName, directive);
```

Comment:
should we have `null` checks for columnName and directive, since both are `nullable`